src/transforms/stdnames.jl

```diff
@@ -0,0 +1,50 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    StdNames(S)
+
+StdNames transform standardizes the column names.
+"""
+struct StdNames <: Stateless
+  spec::Symbol
+end
+
+StdNames() = StdNames(:upper)
+
+isrevertible(::Type{StdNames}) = true
+
+function apply(transform::StdNames, table)  
+  cols = Tables.columns(table)
+  oldnames = Tables.columnnames(cols)
+  spec = transform.spec
+
+  (spec == :camel) && (newnames = map(_camel, map(string, oldnames)))
+  (spec == :snake) && (newnames = map(_snake, map(string, oldnames)))
+  (spec == :upper) && (newnames = map(_upper, map(string, oldnames)))
+
+  newnames = map(Symbol, newnames)
```

Comment:
You don't overwrite variables in Julia because this will lead to type instability often.